src/context/AppProvider/AppContext.tsx

```diff
@@ -138,22 +139,22 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
 
         for (const chain of supportedChains) {
           const adapter = chainAdapterManager.byChain(chain)
+          const { chainNamespace, chainReference } = chainTypeToMainnetChainParts(chain)
+          const chainId = toChainId({ chainNamespace, chainReference })
 
           switch (chain) {
             // TODO: Handle Cosmos ChainType here
             case ChainTypes.Ethereum: {
```

Comment:
what about these?

Reply:
I wasn't going to in this PR, but since you and Kevin both mentioned it: https://github.com/shapeshift/web/pull/1892/commits/effc6bd85de59333b65a003f04274994f77b8b5d